ts/features/itwallet/common/components/ItwCredentialClaim.tsx

```diff
@@ -379,6 +456,54 @@ export const ItwCredentialClaim = ({
           );
           return <PlainTextClaimItem label={claim.label} claim={fiscalCode} />;
         }
+        if (NestedArrayClaim.is(decoded)) {
+          const nestedParsedClaims = decoded.map(item => parseClaims(item));
+          // We render the nested claims as a list if there are multiple items
+          const shouldRenderAsList = nestedParsedClaims.length > 1;
+          if (shouldRenderAsList) {
+            return (
+              <>
+                {nestedParsedClaims.map((singleItemClaims, index) => {
+                  const { summaryLabel, summaryValue } = getNestedItemSummary(
+                    claim.id,
+                    singleItemClaims
+                  );
+                  return (
+                    <Fragment key={`${index}_${claim.id}_${claim.label}`}>
+                      {index > 0 && <Divider />}
+                      <ItwNestedClaimsListItem
+                        itemTitle={summaryValue}
+                        itemClaims={singleItemClaims}
+                        summaryLabel={summaryLabel}
+                        summaryValue={summaryValue}
+                        hidden={hidden}
+                        isPreview={isPreview}
+                        credentialStatus={credentialStatus}
+                        credentialType={credentialType}
+                      />
+                    </Fragment>
+                  );
+                })}
+              </>
+            );
+          } else {
+            const singleItemClaims = nestedParsedClaims[0];
+            return (
+              <>
+                {singleItemClaims.map(nestedClaim => (
+                  <ItwCredentialClaim
+                    key={nestedClaim.id}
+                    claim={nestedClaim}
+                    hidden={hidden}
+                    isPreview={isPreview}
+                    credentialStatus={credentialStatus}
+                    credentialType={credentialType}
+                  />
+                ))}
+              </>
+            );
+          }
+        }
```

Comment:
We could move this to a dedicated component, eg. `ItwCredentialMultiClaim`, that accepts an array of `ParsedCredential`'s claim, and reduce the complexity of this component. What do you think?